src/app/shared-module/bar-plot/bar-plot.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="statistics">
+  <span class="label">{{ viewModel.title }}</span>
+  <div class="chart-group">
+    <div class="y-axis">
+      <span *ngFor="let p of yAxis" class="label chart-label">{{ p }}</span>
+    </div>
+    <div class="chart">
+      <div
+        *ngFor="let bar of viewModel?.dataSequence"
+        class="bar-group"
+        style="--height: {{ calcBarHeight(bar.value) }}%"
+      >
+        <div
+          class="bar"
+          matTooltip="{{ viewModel.tooltipTitle }}: {{ bar.tooltipValue }}"
+          matTooltipPosition="above"
+          matTooltipClass="bar-info-tooltip"
+          matTooltipShowDelay="100"
+          matTooltipHideDelay="100"
+        ></div>
+        <span class="label chart-label x-val">{{ bar.barTitle }}</span>
+      </div>
+    </div>
+  </div>
+  <span class="label d-flex justify-content-end">Hour</span>
```

Comment:
Maybe would be better in BarPlotVM rename title to yAxisTitle, and add xAxisTitle? Than we can use xAxisTitle here

Reply:
That's right, I didn't notice that.

Reply:
Done